tools/data/sthv1/README.md

```diff
@@ -86,7 +86,7 @@ bash extract_flow.sh
 
 ## Step 4. Encode Videos
 
-This part is **optional** if you only want to use RGB frames.
+This part is **optional** if you only want to use RGB videos.
```

Comment:
It is correct, since this step will produce rgb videos. It is optional if you only uses frames.